infrastructure/lib/infrastructure-stack.ts

```diff
@@ -0,0 +1,24 @@
+import {App, Stack, StackProps} from '@aws-cdk/core';
+import {join} from 'path';
+
+import {ServiceHealthSystem} from './health-system';
+
+
+/**
+ * The infrastructure Cfn stack.
+ */
+export class InfrastructureStack extends Stack {
+  constructor(scope: App, id: string, props?: StackProps) {
+    super(scope, id, props);
+
+    const websiteProps = {
+      hostedZone: 'waltersco.co',
+      domainName: 'status.waltersco.co',
+      artifactSourcePath: join(__dirname, '..', '..', 'dist'),
```

Comment:
I think the `artifactSourcePath` should not be an option here. It's not something users need to be able to configure... Instead of exposing all `websiteProps`, just expose `hostedZone` and `domainName` as individual options in `ServiceHealthSystemProps`.